migrations/educandu-2022-06-07-01-add-question-and-answers-to-interactive-media-chapters.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable camelcase, no-console */
+export default class Educandu_2022_06_07_01_add_question_and_answers_to_interactive_media_chapters {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async collectionUp(collectionName) {
+    const result = await this.db.collection(collectionName).updateMany(
+      {},
+      {
+        $set: {
+          'sections.$[sectionElement].content.chapters.$[].question': '[Question]',
+          'sections.$[sectionElement].content.chapters.$[].answers': ['[Answer]'],
+          'sections.$[sectionElement].content.chapters.$[].correctAnswer': 0
+        }
+      },
+      {
+        arrayFilters: [
+          {
+            'sectionElement.type': 'interactive-media',
+            'sectionElement.content': { $ne: null }
+          }
+        ],
+        multi: true
+      }
+    );
+
+    console.log(`Updated ${collectionName}: ${JSON.stringify(result)}`);
+  }
+
+  async collectionDown(collectionName) {
+    const result = await this.db.collection(collectionName).updateMany(
+      {},
+      {
+        $unset: {
+          'sections.$[sectionElement].content.chapters.$[].question': null,
+          'sections.$[sectionElement].content.chapters.$[].answers': null
```

Comment:
Shouldn't we here unset also the `correctAnswer` key?

Reply:
There should indeed, added.